test/test_e2e.rb

```diff
@@ -104,4 +104,14 @@ def test_spec_file_splitting
       "./spec/fast_spec.rb",
     ], queue)
   end
+
+  def test_suite_with_failures_and_fail_fast
+    queue = exec_build("failing_suite", "--fail-fast 1")
+
+    assert queue.build_failed_fast?
+    refute queue.build_successful?
+    assert queue.example_failures.keys.length +
+           queue.non_example_errors.keys.length >=
```

Comment:
I think this assertion is wrong? Don't the number of failures should be less than, or equal to `fail_fast`?

As it currently is, this tests nothing because they **always** will be `>= 1`.